DataConnectors/Qualys VM/host.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Shouldn't there be corresponding changes in run.ps1 to use the new properties defined here? 
What does version = 2.0 signify here?

Reply:
This is from a MSFT documentation reference.

https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json